src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    String draw();
```

Comment:
According to the checklist, interface names should not be verbs. In this case, `Drawable` implies an action. Consider renaming the interface to something like `DrawableFigure` to make it clear that this is an entity capable of being drawn.